modules/oe_theme_content_entity_contact/config/post_updates/00002_create_full_view_display/core.entity_view_display.oe_contact.oe_press.full.yml

```diff
@@ -0,0 +1,167 @@
+langcode: en
```

Comment:
Since we use "Full" view mode for Contact entity instead of "Default" we also need to rename:

- `templates/field/field--oe-contact--oe-press-contact-url--default.html.twig` to `templates/field/field--oe-contact--oe-press-contact-url--full.html.twig` or `field--oe-contact--oe-press-contact-url.html.twig`.
- `templates/oe_contact/oe-contact--default.html.twig` to `templates/oe_contact/oe-contact--full.html.twig` or `templates/oe_contact/oe-contact.html.twig`.

Reply:
I did didn't I?

Reply:
My bad. Yes, these changes exist.